frontend/src/pages/projects/screens/detail/ProjectDetailsComponents.tsx

```diff
@@ -58,8 +56,8 @@ const ProjectDetailsComponents: React.FC = () => {
       ? [
           {
             id: ProjectSectionID.PIPELINES,
-            component: <PipelinesSection />,
-            isEmpty: !pipelinesServer.installed,
+            component: <PipelinesSection setIsPipelinesEmpty={setIsPipelinesEmpty} />,
+            isEmpty: isPipelinesEmpty,
```

Comment:
Andrew's comment eluded to removing `isEmpty` altogether and making it the responsibility of each details section to render the separator as necessary. IMO his suggestion is a clearer approach to having this callback.